shared/fixes/ansible/disable_prelink.yml

```diff
@@ -3,10 +3,22 @@
 # strategy = restrict
 # complexity = low
 # disruption = low
+- name: Verify Prelink is installed
+  command: rpm -q prelink
```

Comment:
do not use `command` unless you absolutely have to in Ansible, in this case this can be done with the package command and it will work everywhere even on dpkg distros

Reply:
I believe that the package command requires a state that will either install or remove the package.  The package command could be used to uninstall prelink making the subsequent file edit obsolete.